config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,18 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// OpenStackPlatformStatus holds the current status of the OpenStack infrastructure provider.
+type OpenStackPlatformStatus struct {
+	// ApiIntIp is an IP address managed by the OpenStack provider
+	// backing the internal `api-int` record.
+	ApiIntIp string `json:"apiIntIp,omitempty`
```

Comment:
For the openapi spec, field name reference in the godoc must start with lowercase, like `apiIntIp`. I'm also suspecting `IP` and `API` should be caps but not sure.

Reply:
Thanks, fixed!

Yeah I didn't find anything about casing in the openapi spec, but at least go's code review guidelines are pretty clear it should be capitalised:

https://github.com/golang/go/wiki/CodeReviewComments#initialisms